tests/test_rds.py

```diff
@@ -800,6 +804,143 @@ def test_rds_health_events_filter(self):
         self.assertEqual(len(resources), 0)
 
 
+class TestRDSParameterGroupFilter(BaseTest):
+    """ This test assumes two parameter groups have been created and
+        assigned to two different RDS instances.
+    """
+    DEFAULT_REGION = 'us-east-1'  # N. Virginia
+    REQUIRED_PARAMETERGROUPS = [
+        'rds-pg-group-a',
+        'rds-pg-group-b',
+    ]
+    RECORD = False
+    EXAMPLE_YAML_STRING_SHOULD_EQ = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: log_destination
+                op: eq
+                value: stderr
+    '''
+    EXAMPLE_YAML_STRING_SHOULD_NOT_EQ = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: log_destination
+                op: ne
+                value: s3
+    '''
+
+    EXAMPLE_YAML_INT_DEFAULT = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: autovacuum_multixact_freeze_max_age
+                op: eq
+                value: 7
+                default: 7
+    '''
+    EXAMPLE_YAML_INT_SHOULD_EQ = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: full_page_writes
+                op: eq
+                value: 1
+    '''
+    EXAMPLE_YAML_INT_SHOULD_GT = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: full_page_writes
+                op: gt
+                value: 0
+    '''
+    EXAMPLE_YAML_INT_SHOULD_NOT_LT = '''
+        policies:
+          - name: filter-rds-by-paramgroup
+            resource: rds
+            filters:
+              - type: db-parameter
+                key: full_page_writes
+                op: lt
+                value: 1
+    '''
+
+    def _get_test_policy(self, name, yaml_doc, record=False):
+        if record:
+            logger.warn("%s is RECORDING" % self.__class__.__name__)
```

Comment:
this has come up before for others, please don't hack in separate support for recording switches, it really is just changing 8 chars or less at the moment to switch, I'd prefer to switch the whole set generically rather than per test class implementations.